src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC_PerfInitPage.js

```diff
@@ -160,124 +145,254 @@ class CJ4_FMC_PerfInitPage {
     }
 	static ShowPage6(fmc) { //TAKEOFF REF Page 1
 		fmc.clearDisplay();
+		let originIdent = "";
+        let origin = fmc.flightPlanManager.getOrigin();
+        if (origin) {
+            originIdent = origin.ident;
+        }
+		let depRunwayDesignation = "";
+		let depRunwayDirection = "";
+        let depRunwayElevation = "";
+        let depRunwayLength = "";
+        let depRunwayMod = "";
+        //added CWB method for runway designation
+        //let selectedRunway = fmc.flightPlanManager.getDepartureRunway();
+        let depRunwayOutput = "";
+        if (fmc.flightPlanManager.getDepartureRunway()) {
```

Comment:
Is this code the same as the other spot that gets the runway designation? Can we stick this somewhere where both can use it, if that's the case?

Reply:
Yea, so I could make a method in CJ4_FMS.js I guess and reference it? I was trying to avoid editing that file, but I guess I should do that.